insights/core/exceptions.py

```diff
@@ -1,3 +1,49 @@
+class AutologyError(Exception):
```

Comment:
@ryan-blakley - do you think it's necessary to add a doc-entry/index for this `insights.core.exceptions` module in the `docs`?